### PR TITLE
Change CMC api from v1 to pro version

### PR DESCRIPTION
**Pull request by IvanIvanoff**
#### Summary

[//]: # (#### Related PRs)

[//]: # (#### Additional deploy notes)

[//]: # (#### Screenshots)

### CODE REVIEW

File: lib/sanbase/external_services/coinmarketcap2/ticker.ex [hunk 1]
```diff
@@ -77,13 +82,64 @@ defmodule Sanbase.ExternalServices.Coinmarketcap.Ticker2 do
...
-  def parse_json(json) do
-    json
-    |> Poison.decode!(as: [%Ticker{}])
-    |> Stream.filter(fn %Ticker{last_updated: last_updated} -> last_updated end)
-    |> Enum.map(&make_timestamp_integer/1)
+  @spec parse_json(String.t()) :: [%Ticker{}] | no_return
+  defp parse_json(json) do
+    %{"data" => data} =
+      json
+      |> Poison.decode!()
```
Comment (tspenov) [inline lib/sanbase/external_services/coinmarketcap2/ticker.ex]: we can use `Jason` since we are not using `as` opt ?

**Comment by IvanIvanoff** [comment]
@tspenov it's two different modules because when we migrate the prices I'll just delete all the old modules.

**Comment by tspenov** [comment]
There is some random failing test `Sanbase.Github.SchedulerTest` - its happening more than once, although replay fixes it

**Comment by tspenov** [comment]
> @tspenov it's two different modules because when we migrate the prices I'll just delete all the old modules.

Just seems to me that the difference here is very small related to the duplicated code and I needed to compare both side to side to see what is the different thing. But, ok, lets keep it as it is

**Comment by tspenov** [comment]
> @tspenov it's two different modules because when we migrate the prices I'll just delete all the old modules.

Just seems to me that the difference here is very small related to the duplicated code and I needed to compare both side to side to see what is the different thing. But, ok, lets keep it as it is

**Pull request merged**